css/properties/grid-template-columns.json

```diff
@@ -410,66 +248,34 @@
                   "version_removed": "57",
                   "notes": "<a href='https://developer.mozilla.org/docs/Web/CSS/calc'><code>calc()</code></a> doesn't work in <code>repeat()</code> (see <a href='https://bugzil.la/1350069'>bug 1350069</a>).",
                   "partial_implementation": true
-                },
-                {
-                  "version_added": "40",
-                  "version_removed": "59",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "layout.css.grid.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
                 }
               ],
               "firefox_android": [
                 {
-                  "version_added": "52"
+                  "version_added": "76"
```

Comment:
Here's the first irregular change to look extra carefully at.

Reply:
Looking at the data, this seems like a reasonable change to me.  The commit history indicates that changes were made to Firefox desktop that should have been made to FxA as well, but weren't.  I can't think of a reason that the data should vary between the two.